include/mbedtls/ssl.h

```diff
@@ -2533,13 +2533,19 @@ void mbedtls_ssl_conf_cert_req_ca_list( mbedtls_ssl_config *conf,
 #if defined(MBEDTLS_SSL_MAX_FRAGMENT_LENGTH)
 /**
  * \brief          Set the maximum fragment length to emit and/or negotiate
- *                 (Default: the smaller of MBEDTLS_SSL_IN_CONTENT_LEN and
+ *                 (Typical: the smaller of MBEDTLS_SSL_IN_CONTENT_LEN and
  *                 MBEDTLS_SSL_OUT_CONTENT_LEN, usually 2^14 bytes)
  *                 (Server: set maximum fragment length to emit,
- *                 usually negotiated by the client during handshake
+ *                 usually negotiated by the client during handshake)
```

Comment:
Minor and pre-existing: I'm not a native, but `negotiated by [single entity]` doesn't sound right to me. 

I'd prefer to keep this overview section short, and provide details further below. What about:
```
   (Server: set the maximum fragment length to emit)
   (Client: set the maximum fragment length to emit and to communicate to the server)

\note On the server, the maximum fragment length extension doesn't provide a means to communicate
      the server's message size constraints to the client. If you're running a constrained server,
      you must ensure out of band that the client will not send messages larger than what the
      server can handle.
      On the client, the client will communicate its limitation to the server.
```

Feel free to elaborate on this, potentially drawing from https://tls.mbed.org/kb/how-to/controlling_package_size which explains everything in much more detail. Maybe we could even link the KB article here?